posts/2024-03-22-march-convergence-of-ai-blockchain-privacy.md

```diff
@@ -26,18 +26,13 @@ An overview of the need for decentralised and privacy-preserving artificial inte
 After the recent boom in the generative artificial intelligence (AI) space, several critics have questioned the convergence of AI and web3. There is also a growing misconception that web3 does not have native use cases within the AI space. Indeed, some are popularising an unfounded argument that web3 is being forced into the AI space. This blog post will explore the need for an intersection between these cutting-edge technologies and some potential use cases where we can implement a technology like [Waku](https://waku.org/) in AI-related areas.
 
 
-## The need for an intersection between AI and Web3
+## The need for an intersection between AI and web3
 
 I have always believed that web3 is not 'just' about blockchains but extends to the broader idealogy of building censorship-resistant infrastructure with privacy-preserving technologies.
 
-We often only understand the need for such ethos once we get exposed to the problems that hinder innovation through censorship. A perfect example of this scenario is the following public announcement from the Indian government below:
+We often only understand the need for such ethos once we get exposed to the problems that hinder innovation through censorship. A perfect example of this scenario is a recent [statement from the Government of India's IT Minister](https://timesofindia.indiatimes.com/gadgets-news/government-to-tech-companies-take-permission-before-launching-ai-models-in-india/articleshow/108167769.cms), Rajeev Chandrasekhar.
```

Comment:
can we avoid the name of the person? maybe we could just say 'government official' or 'IT Minister'?